build.gradle.kts

```diff
@@ -59,20 +59,18 @@ kotlin {
 
         jvm().compilations["main"].defaultSourceSet {
             dependencies {
-                implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
-                implementation("io.github.microutils:kotlin-logging:1.7.9")
-                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.2")
+                implementation("io.github.microutils:kotlin-logging:2.0.10")
```

Comment:
`implementation("io.github.microutils:kotlin-logging:2.0.10")` is also declared on `commonMain`.
Is this sentence necessary?

Reply:
no, good catch.